test/TestHelpers.cs

```diff
@@ -298,13 +298,13 @@ private static List<string> GetLogs_UnhandledActivityException(string messageId,
             {
                 $"{messageId}: Function '{orchestratorFunctionNames[0]} ({FunctionType.Orchestrator})', version '' scheduled. Reason: NewInstance. IsReplay: False.",
                 $"{messageId}: Function '{orchestratorFunctionNames[0]} ({FunctionType.Orchestrator})', version '' started. IsReplay: False. Input: \"Kah-BOOOOM!!!\"",
-                $"{messageId}: Function '{activityFunctionName} ({FunctionType.Activity})', version '' scheduled. Reason: Throw. IsReplay: False.",
+                $"{messageId}: Function '{activityFunctionName} ({FunctionType.Activity})', version '' scheduled. Reason: ThrowOrchestrator. IsReplay: False.",
                 $"{messageId}: Function '{orchestratorFunctionNames[0]} ({FunctionType.Orchestrator})', version '' awaited. IsReplay: False.",
                 $"{messageId}: Function '{activityFunctionName} ({FunctionType.Activity})', version '' started. IsReplay: False. Input: [\"Kah-BOOOOM!!!\"]",
-                $"{messageId}: Function '{activityFunctionName} ({FunctionType.Activity})', version '' failed with an error. Reason: Microsoft.Azure.WebJobs.Host.FunctionInvocationException",
```

Comment:
>Microsoft.Azure.WebJobs.Host.FunctionInvocationException [](start = 133, length = 56)

The 'Reason: Microsoft.Azure.WebJobs.Host.FunctionInvocationException' entry in all these logs is also giving me a quick clue that the problem is a FunctionInvocationException, vs FunctionTimeoutException. In long and complex execution workflows, it may help quickly identify issues. We lose this visibility with this change. Do you think we really don't need to surface this information?

Reply:
I've never heard of `FunctionTimeoutException`. If a function takes to long to execute, WebJobs will recycle the host. But even if we do have other exception types, I'm only ever stripping `FunctionInvocationException`, so you should still be able to figure out what's going on, right? In any case, customers have never needed to deal with these internal WebJobs exceptions, so I didn't want to surface them here.

---
In reply to: [181524809](https://github.com/Azure/azure-functions-durable-extension/pull/250#discussion_r181524809) [](ancestors = 181524809)